### PR TITLE
refactor(console): support tiny pagination variant

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

refactor(console): support tiny pagination variant
image

## Testing

image

### CODE REVIEW

File: packages/console/src/components/Pagination/index.module.scss [hunk 1]
```diff
@@ -4,52 +4,75 @@
...
+        }
       }
     }
+  }
 
-    &.disabled {
-      .button {
-        cursor: not-allowed;
-        background: var(--color-neutral-95);
+  &.default {
```
Comment (simeng-li) [inline packages/console/src/components/Pagination/index.module.scss:55]: should use tiny to overwrite the styling? Is the default class name necessary?

Comment (xiaoyijun) [reply packages/console/src/components/Pagination/index.module.scss]: updated.

File: packages/console/src/components/Pagination/index.tsx [hunk 2]
```diff
@@ -15,10 +16,18 @@ type Props = {
   totalCount?: number;
   pageSize: number;
   className?: string;
+  isTinyStyle?: boolean;
```
Comment (charIeszhao) [inline packages/console/src/components/Pagination/index.tsx:19]: `type/mode?: 'normal' | 'pico';`

File: packages/console/src/components/Pagination/index.tsx [hunk 3]
```diff
@@ -37,8 +46,15 @@ const Pagination = ({ pageIndex, totalCount, pageSize, className, onChange }: Pr
   const min = (pageIndex - 1) * pageSize + 1;
   const max = Math.min(pageIndex * pageSize, cachedTotalCount);
 
+  const pageRangeConfig = conditional(
+    isTinyStyle && {
+      pageRangeDisplayed: -1,
+      marginPagesDisplayed: 0,
+    }
+  );
```
Comment (charIeszhao) [inline packages/console/src/components/Pagination/index.tsx:54]: nit: Personally I would just apply these props on the JSX element.
```ts
const isPicoMode = mode === 'pico';

<ReactPaginate
 ...
 pageRangeDisplayed={conditional(isPicoMode && -1)}
 marginPagesDisplayed={conditional(isPicoMode && 0)}
/>
```

Comment (xiaoyijun) [reply packages/console/src/components/Pagination/index.tsx]: updated with ternary operation for the conditional will cause an error.
image

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**